testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/messaging/mgmt/JMSBridgeManagementTestCase.java

```diff
@@ -0,0 +1,360 @@
+                TestSuiteEnvironment.getServerPort(),CONTAINER);
+        for(ServerSetupTask t:this.tasks){
+            this.tasksToClean.add(t);
+            t.setup(this.managementClient, CONTAINER);
+        }
+    }
+
+    @After
+    public void cleanup(){
+        try{
+            for(ServerSetupTask sst:this.tasksToClean){
+                sst.tearDown(this.managementClient, CONTAINER);
+            }
+        }catch(Exception e){
+            
+        }
+        this.tasksToClean.clear();
+        try{
+            reload();
+        }catch(Exception e){
+            
+        }
+    }
+
+    @Deployment(name = DEPLOYMENT_NAME, managed = false)
+    @TargetsContainer(CONTAINER)
+    public static Archive<?> getDeployment() {
+        JavaArchive ja = ShrinkWrap.create(JavaArchive.class, "dummy.jar");
+        ja.addPackage(JMSBridgeManagementTestCase.class.getPackage());
+        return ja;
+    }
+    
+    @Test
+    public void testStopStart() throws Exception {
+        //actually even checking state seems enough, but lets try a circle.
+        reload();
+        final ModelControllerClient client = this.managementClient.getControllerClient();
+        ModelNode operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        ModelNode result = client.execute(operation);
+        Assert.assertTrue("Bridge should start!",result.get(RESULT).asBoolean());
+        
+        operation = Util.createOperation(STOP, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        result = client.execute(operation);
+        operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        result = client.execute(operation);
+        Assert.assertFalse("Bridge should stop!",result.get(RESULT).asBoolean());
+
+        operation = Util.createOperation(START, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        result = client.execute(operation);
+        operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        result = client.execute(operation);
+        Assert.assertTrue("Bridge should start!",result.get(RESULT).asBoolean());
+    }
+
+    @Override
+    protected ModelControllerClient getModelControllerClient() {
+        return this.managementClient.getControllerClient();
+    }
+
+    public void reload() throws Exception {
+        executeReloadAndWaitForCompletion(this.managementClient.getControllerClient(), 100000);
+    }
+    
+    
+    static class SetupConnectionFactory implements ServerSetupTask {
```

Comment:
org.jboss.as.test.integration.common.jms.JMSOperations could be used to create the connection factory